### ISSUE TITLE
Mixed chart type

### ISSUE THREAD

**Issue by thienanle** [rma-consulting/react-easy-chart#86]
Would be great to support this, to do something like:

![image](https://cloud.githubusercontent.com/assets/874402/19951667/bc053044-a1b3-11e6-91e7-e4a09dfa9450.png)

**Comment by harunhasdal** [comment]
@thienanle would this be similar to what you are asking? http://rma-consulting.github.io/react-easy-chart/bar-chart/index.html#bar_and_line

**Comment by thienanle** [comment]
@harunhasdal Yes that would serve the purpose of the above chart. However, it would be nice to have more flexibility, as other x-y type charts could also benefit from this (for example, area and line, scatter and line).

**Comment by harunhasdal** [comment]
@thienanle I agree. This would however mean that we should expose more components in order to have a flexible API. We had a discussion about this earlier, it is always a tradeoff between simple api vs flexibility. If you have a workable API in mind, we can look into implementing it; we also welcome PRs. 

FYI below is the sort of API we had in mind

```
<Graph data=...>
  <XAxis .../>
  <YAxis .../>
  <Series>
    <BarChart .../>
    <LineChart.../>
  </Series>
</Graph>
```
